lib/elixir/test/elixir/fixtures/calendar/holocene.exs

```diff
@@ -12,33 +14,63 @@ defmodule Calendar.Holocene do
                    microsecond: microsecond, calendar: __MODULE__}
   end
 
+  @impl true
   def date_to_string(year, month, day) do
     "#{year}-#{month}-#{day} (HE)"
   end
 
+  @impl true
   def naive_datetime_to_string(year, month, day, hour, minute, second, microsecond) do
-    "#{year}-#{month}-#{day} #{Calendar.ISO.time_to_string(hour, minute, second, microsecond)} (HE)"
+    "#{year}-#{month}-#{day}" <>
+      Calendar.ISO.time_to_string(hour, minute, second, microsecond) <> " (HE)"
+  end
+
+  @impl true
+  def datetime_to_string(year, month, day,
+                         hour, minute, second, microsecond,
+                         _time_zone, zone_abbr, _utc_offset, _std_offset) do
+    "#{year}-#{month}-#{day}" <>
+      Calendar.ISO.time_to_string(hour, minute, second, microsecond) <> " #{zone_abbr} (HE)"
```

Comment:
it doesn't matter for a test calendar, but perhaps it would be useful for folks to make `Calendar.ISO.{offset_to_string, zone_to_string}` public? This way we could just delegate, similar to how it is in ISO: https://github.com/elixir-lang/elixir/blob/v1.5.0-rc.1/lib/elixir/lib/calendar/iso.ex#L268:L269. But yeah, probably better to wait with it until there's a real use case.